### DISCUSSION TITLE
Introducing the New GitLens Home View in GitLens 16

### DISCUSSION THREAD

**Discussion by justinrobots** [gitkraken/vscode-gitlens#3721] [Announcements]
In GitLens 16, we’ve reimagined the GitLens Home View (accessible via the GitLens icon in the VS Code activity bar) to better guide you through your development workflows.

<img width="1188" alt="old-vs-new" src="https://github.com/user-attachments/assets/493dc2b0-e326-4829-8641-a82ca19cb378">

### The new Home View is designed to help you answer three key questions throughout your day:

- [What am I actively working on now?](#what-am-i-actively-working-on-now)
- [What do I need to work on next?](#what-do-i-need-to-work-on-next)
- [What have I worked on recently?](#what-have-i-worked-on-recently)

<h2 id="what-am-i-actively-working-on-now">What am I actively working on now?</h2>
The **ACTIVE** section shows your current repository and branch, alongside the repository state and actions for syncing (push, pull, fetch), switching repos/branches, and viewing working directory changes.

If your branch is tied to a pull request or issue, GitLens surfaces those details so you can check PR status, continue reviews, or revisit issue specs. GitLens will also guide you to start a PR from your current branch if it can't find one.

> 💡 Connect integrations to view pull request and issue information in GitLens

<p align="center"><img width="675" alt="active" src="https://github.com/user-attachments/assets/7e4931e7-2f0d-4c4b-9dac-0f3aad81e87f"></p>

<h2 id="what-do-i-need-to-work-on-next">What do I need to work on next?</h2>
The **LAUNCHPAD** section unblocks you and your team by showing pull requests that need your review, are blocked, or are ready to merge. Quickly jump to PRs to review, comment, or merge.

With the new **Start work** action, you can easily begin a new branch or worktree, or generate one from an existing issue.

<p align="center"><img width="675" alt="launchpad" src="https://github.com/user-attachments/assets/761855db-96aa-4991-a74c-48ba6952fe2f"></p>

<h2 id="what-have-i-worked-on-recently">What have I worked on recently?</h2>
The **RECENT** section lets you quickly return to previous work by showing recent branches, worktrees, and PRs with activity for your chosen timeframe.

<p align="center"><img width="675" alt="recent" src="https://github.com/user-attachments/assets/de64ae88-4cf1-44b8-8f05-42a536252309"></p>

---
We’d love your feedback on this new experience—try it out and let us know what you think!

**Comment by fmabid** [comment]
I was using GitLens's classical look. Now, after switching to the new version, I am a little lost.

I have two questions:
1. How do we merge a remote into the current branch locally?
2. Where do find/get the list of all remote branches?

**Reply by justinrobots** [reply]
Hi @fmabid the `Remotes` view was moved into a consolidated `GitLens` view in Source Control. It can be restored to its original position from the menu in that new view. More about that change in this [discussion](https://github.com/gitkraken/vscode-gitlens/discussions/3762).

Does locating the `Remotes` view answer the first part of your question?

**Reply by fmabid** [reply]
@justinrobots yeah, thanks.

**Reply by SoeAung95** [reply]
$ git remote -v
# Verify new remote URL
> origin  https://github.com/OWNER/REPOSITORY.git (fetch)
> origin  https://github.com/OWNER/REPOSITORY.git (push)

**Comment by evenkov** [comment]
The new Home View is as everything new a bit confusing. So I have lost in working in Worktrees. Here is my confusion:
I have two worktree(`default` - working branch, and second `quick fix`). On new Home View when I'm working on `default` I can see the `quick fix` one. There is no option from here to open it in new window by `Alt`+`Click`.



<img width="455" alt="HomeView-QuickFIx" src="https://github.com/user-attachments/assets/d5f7d30b-1fac-4b31-90d1-711a1030dd28" style="max-width:100%">



After I switch to worktree `quick fix` in Home View I cannot return to my `default` one. It shows me only branch for default worktree


<img width="455" alt="HomeView-QuickFIx" src="https://github.com/user-attachments/assets/136c27d3-9275-487c-895f-f0004016b4f3" style="max-width:100%">




So the only way to switch back to `default` worktree is from Command Palette ->GitLens: Git Open Worktree...

The other way I can switch to worktree is from GitLens tab in Source Control. There is a option to open Worktree in New Window with `Alt `+ `Click`

Maybe that's concept in new Home View or?

EDIT:
So after all day working on several worktree I think I understood: the default is my branch work directory and all other are `copies`/`worktrees`. So it is correct shown as branch, not worktree. The problem is than if I open some worktree in same window of **VSCode** then I cannot return/`switch to branch` to `default` one!

**Reply by justinrobots** [reply]
Hi @evenkov thanks for the feedback!

1. GitLens views like `Worktrees` were moved into a consolidated `GitLens` view in Source Control. You can easily restore them to their previous locations or move them wherever you want. More on that change in this [discussion](https://github.com/gitkraken/vscode-gitlens/discussions/3762).
2. The default worktree showing as a branch in the new Home instead of a worktree is a bug. We're on it! 
 
https://github.com/gitkraken/vscode-gitlens/issues/3764

**Reply by evenkov** [reply]
Thanks!
I'm getting used with `Home View` and in general I like the idea. 
I after I wrote this I found [discussion](https://github.com/gitkraken/vscode-gitlens/discussions/3762) and now used `GitLens` view in Source Control. The idea to combine all this views in one - `GitLens` is very good and I think most of us will like it!

**Comment by jgnolstudio** [comment]
> In GitLens 16, we’ve reimagined the GitLens Home View (accessible via the GitLens icon in the VS Code activity bar) to better guide you through your development workflows. This updated experience is now available in the GitLens pre-release and can be enabled in Settings with: `"gitlens.home.preview.enabled": true`.
> 
> <img alt="old-vs-new" width="1188" src="https://private-user-images.githubusercontent.com/3358707/384075468-493dc2b0-e326-4829-8641-a82ca19cb378.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4Mjk3NzQsIm5iZiI6MTczMjgyOTQ3NCwicGF0aCI6Ii8zMzU4NzA3LzM4NDA3NTQ2OC00OTNkYzJiMC1lMzI2LTQ4MjktODY0MS1hODJjYTE5Y2IzNzgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjhUMjEzMTE0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NmE3MDBkYjQ5MzkzYTlhZWY1OWRiYWMwYmVhMTIxN2M3OGE1Y2FkODA4NWQyMjdmNzBjNmNlYmE2ZTA5OWFlZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.RHu2NX_V-Bb8H3L5InMgq0aOTO5n4wymn49a6dhjgvI">
> ### The new Home View is designed to help you answer three key questions throughout your day:
> * [What am I actively working on now?](#what-am-i-actively-working-on-now)
> * [What do I need to work on next?](#what-do-i-need-to-work-on-next)
> * [What have I worked on recently?](#what-have-i-worked-on-recently)
> 
> ## What am I actively working on now?
> The **ACTIVE** section shows your current repository and branch, alongside the repository state and actions for syncing (push, pull, fetch), switching repos/branches, and viewing working directory changes.
> If your branch is tied to a pull request or issue, GitLens surfaces those details so you can check PR status, continue reviews, or revisit issue specs. GitLens will also guide you to start a PR from your current branch if it can't find one.
> 
> > 💡 Connect integrations to view pull request and issue information in GitLens
> 
> <img alt="active" width="675" src="https://private-user-images.githubusercontent.com/3358707/384076753-7e4931e7-2f0d-4c4b-9dac-0f3aad81e87f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4Mjk3NzQsIm5iZiI6MTczMjgyOTQ3NCwicGF0aCI6Ii8zMzU4NzA3LzM4NDA3Njc1My03ZTQ5MzFlNy0yZjBkLTRjNGItOWRhYy0wZjNhYWQ4MWU4N2YucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjhUMjEzMTE0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTFjYWExNjJhN2JjYjg4NzFmYWE0YWQ0ZGVhZjBkMDQzZjI2ZDJkMTE1OWZjZTljNmYyNWRmYTRmMjFmNTQzZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.CfXEyZps-tLf0tfeR1qS2j_O2uPcWXxs3I2YpvYAskE">
> 
> ## What do I need to work on next?
> The **LAUNCHPAD** section unblocks you and your team by showing pull requests that need your review, are blocked, or are ready to merge. Quickly jump to PRs to review, comment, or merge.
> With the new **Start work** action, you can easily begin a new branch or worktree, or generate one from an existing issue.
> 
> <img alt="launchpad" width="675" src="https://private-user-images.githubusercontent.com/3358707/384075743-761855db-96aa-4991-a74c-48ba6952fe2f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4Mjk3NzQsIm5iZiI6MTczMjgyOTQ3NCwicGF0aCI6Ii8zMzU4NzA3LzM4NDA3NTc0My03NjE4NTVkYi05NmFhLTQ5OTEtYTc0Yy00OGJhNjk1MmZlMmYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjhUMjEzMTE0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjExN2EzNTVjMTg4OTJmZWI4YmYwN2Q5NWE3YmI0MmVjY2RmYWU4MzUxYzRkNGMwOWNiYmMzYzkyOGE5NDk2ZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.yQucgnk1nJm6uiHI2JzbbKuHHwIC_myarnzYwoAlKw4">
> 
> ## What have I worked on recently?
> The **RECENT** section lets you quickly return to previous work by showing recent branches, worktrees, and PRs with activity for your chosen timeframe.
> <img alt="recent" width="675" src="https://private-user-images.githubusercontent.com/3358707/384077073-de64ae88-4cf1-44b8-8f05-42a536252309.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4Mjk3NzQsIm5iZiI6MTczMjgyOTQ3NCwicGF0aCI6Ii8zMzU4NzA3LzM4NDA3NzA3My1kZTY0YWU4OC00Y2YxLTQ0YjgtOGYwNS00MmE1MzYyNTIzMDkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjhUMjEzMTE0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZWRmNTc5MzlmYzJkMTE1YTQ2ZWQyZTg3ZWE3ZTUzZjdhYjExZTM2ZmU0ODM2Mzk3MmI4NWI2YWNhZGVkNTBkYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.9SC3bTdC4ZKOdmrkvRlAmzmZMH-7w0cRzAny5XXgW1w">
> 
> We’d love your feedback on this new experience—try it out and let us know what you think!

**Comment by soumyd** [comment]
Merci

**Comment by glebbash** [comment]
This panel is opening everytime I open a repository with GitPod. 
![image](https://github.com/user-attachments/assets/d3215580-0d40-4f53-9b1d-351dd2291734)

I have GitLens view disabled in the side bar but it doesn't care and still shows up every time.
![image](https://github.com/user-attachments/assets/9d49fffd-a3dc-49b4-9b2d-706bdfeb6233)

How do I disable this?

**Comment by irajsava** [comment]
- [ ] ##

**Comment by zhaoccqq** [comment]
Bob

**Comment by g8ecj** [comment]
Lost Search and Compare in 16.1.1
I can get the menu item in the old home menu but it does nothing when I click on it.  I can't find it in the new layout at all.
Totally screwed my bug tracking :(

**Comment by irajsava** [comment]
No no

در تاریخ شنبه ۲۱ دسامبر ۲۰۲۴،‏ ۱۷:۳۱ 504 ***@***.***> نوشت:

**Comment by octavio12345300** [comment]
list

**Comment by emike108** [comment]
Hi, I am seeing a discrepancy between what I am seeing from the extension between VSCode and VSCode - Insiders. 

When I look at the Git Blame in stable VSCode I see a few ways to link to the PR that created that line.
When I try to do the same in VSCode - Insiders I don't see that.
It seems like in VSCode - Insiders it doesn't pick up the Git AutoLinks. Also my GitHub avatar isn't picked up? I should be successfully connected to my GitHub account on both.

Is that expected? How can I fix that?

VSCode:
<img width="1547" alt="image" src="https://github.com/user-attachments/assets/c5dfaa50-6f45-45ea-af18-31132f59582f" />

VSCode - Insiders:
<img width="1586" alt="image" src="https://github.com/user-attachments/assets/03046807-b441-40bd-8437-ff6f468c3c25" />

**Reply by emike108** [reply]
@justinrobots

**Reply by eamodio** [reply]
The first thing I would check is if you have custom remote integrations configured in your settings that aren't configured in Insiders. If that doesn't help, please open a new issue, as this has nothing to do with the new Home view, with more detail as to the issue you are experiencing and/or the steps to reproduce it?

Also, if this issue is reproducible, can you please enable debug logging by running from _GitLens: Enable Debug Logging_ from the Command Palette (<kbd>F1</kbd> or <kbd>ctrl/cmd</kbd>+<kbd>shift</kbd>+<kbd>p</kbd>). This will enable logging to the _GitLens_ and _GitLens (Git)_ channel in the _Output_ pane.

Once enabled, please reproduce the issue, and attach the log from both the _GitLens_ and _GitLens (Git)_ channels.

Thank you!

**Comment by hunter93** [comment]
Hey! Just started on the Pro version of GitLens and one thing I'm most focused on using is the ability to jump to the PR for a certain commit.

However, for some reason, GitLens is not updating my integrations in VS Code Insiders. I will attempt on VS Code as well to see if it works there.

As you can see from the screenshots provided, in VS Code Insiders, GitLens is not showing as connected to GitHub, and also when I hit the button to "Synchronize Status", it appears to do nothing.

However, when I go to the GitLens site, it shows I am connected to my Jira and GitHub accounts and I've tried quitting out of VS Code Insiders and restarting the application once or twice as well.

![Screenshot 2025-04-02 at 3 00 22 PM](https://github.com/user-attachments/assets/f6964fa5-4993-406d-8f9e-3676ca28aaeb)
<img width="1200" alt="Screenshot 2025-04-02 at 3 00 31 PM" src="https://github.com/user-attachments/assets/23e9ceb6-2341-49e9-a498-ee0bb1184095" />

**Comment by SilkePilon** [comment]
Hi, found an issue with the new ui.

When hovering over the profile picture of the user the user name gets blocked by the "Switch to branch" button

<img width="238" alt="{6B292440-6C39-4167-9042-861E32DA06B1}" src="https://github.com/user-attachments/assets/d1019f46-92a7-4ce9-a5c4-08f9fed87319" />

hope this helps

**Reply by justinrobots** [reply]
Thanks for the feedback @SilkePilon !

**Comment by hungdoansy** [comment]
Hi, is there a way to switch back to the previous view? All I want to File History and Line History back

**Reply by justinrobots** [reply]
Hi @hungdoansy , thanks for the feedback. You can still find the File and Line History views in GitLens Inspect. You could be missing them if GitLens Inspect is hidden in your sidebar.

<img width="454" alt="image" src="https://github.com/user-attachments/assets/7afc361c-5fa4-472b-92f4-d8d28c1f44ca" />

**Comment by ghost** [comment]
> In GitLens 16, we’ve reimagined the GitLens Home View (accessible via the GitLens icon in the VS Code activity bar) to better guide you through your development workflows.
> 
> <img alt="old-vs-new" width="1188" src="https://private-user-images.githubusercontent.com/3358707/384075468-493dc2b0-e326-4829-8641-a82ca19cb378.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY4MDc3NDEsIm5iZiI6MTc1NjgwNzQ0MSwicGF0aCI6Ii8zMzU4NzA3LzM4NDA3NTQ2OC00OTNkYzJiMC1lMzI2LTQ4MjktODY0MS1hODJjYTE5Y2IzNzgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDkwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA5MDJUMTAwNDAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9M2RmOTExNDI3NWE3Y2FkMjM0ZjU0YjhjZmEwMzViOWVkYjM2YTdmYWE4MzRlZGExMTkyZDE4YTRhNWVjZDRjYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.17x-qgt2kNBFOZQ2PeQRipxXFo-ibxCpz2egurGjBZo">
> ### The new Home View is designed to help you answer three key questions throughout your day:
> * [What am I actively working on now?](#what-am-i-actively-working-on-now)
> * [What do I need to work on next?](#what-do-i-need-to-work-on-next)
> * [What have I worked on recently?](#what-have-i-worked-on-recently)
> 
> ## What am I actively working on now?
> The **ACTIVE** section shows your current repository and branch, alongside the repository state and actions for syncing (push, pull, fetch), switching repos/branches, and viewing working directory changes.
> If your branch is tied to a pull request or issue, GitLens surfaces those details so you can check PR status, continue reviews, or revisit issue specs. GitLens will also guide you to start a PR from your current branch if it can't find one.
> 
> > 💡 Connect integrations to view pull request and issue information in GitLens
> 
> <img alt="active" width="675" src="https://private-user-images.githubusercontent.com/3358707/384076753-7e4931e7-2f0d-4c4b-9dac-0f3aad81e87f.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY4MDc3NDEsIm5iZiI6MTc1NjgwNzQ0MSwicGF0aCI6Ii8zMzU4NzA3LzM4NDA3Njc1My03ZTQ5MzFlNy0yZjBkLTRjNGItOWRhYy0wZjNhYWQ4MWU4N2YucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDkwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA5MDJUMTAwNDAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjUwMmJmNGNhNzVmZTcwN2Y2ZmIxMGI4NjYwYTk0NzllYzViMDUzMzE1NTViYTg3YTc2NmFhMmEwZTI2ZTc4MCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.1bsArNvqSgg_7Gvm-TJG9S-Ij63EWXk5SQrfeYqx6gg">
> 
> ## What do I need to work on next?
> The **LAUNCHPAD** section unblocks you and your team by showing pull requests that need your review, are blocked, or are ready to merge. Quickly jump to PRs to review, comment, or merge.
> With the new **Start work** action, you can easily begin a new branch or worktree, or generate one from an existing issue.
> 
> <img alt="launchpad" width="675" src="https://private-user-images.githubusercontent.com/3358707/384075743-761855db-96aa-4991-a74c-48ba6952fe2f.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY4MDc3NDEsIm5iZiI6MTc1NjgwNzQ0MSwicGF0aCI6Ii8zMzU4NzA3LzM4NDA3NTc0My03NjE4NTVkYi05NmFhLTQ5OTEtYTc0Yy00OGJhNjk1MmZlMmYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDkwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA5MDJUMTAwNDAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YTUyZjUzYjk4MDRhYmRhM2FkNzdlNzE2ZWY0YmI0MjM2OGMyYWI4Y2Y1NjcyMDg2OTIyNWYxYTkxM2RlZTRiMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.QEwjSF2h4xelufAGS5iuDYAFMRPpCdzCJcBfwDeorkY">
> 
> ## What have I worked on recently?
> The **RECENT** section lets you quickly return to previous work by showing recent branches, worktrees, and PRs with activity for your chosen timeframe.
> <img alt="recent" width="675" src="https://private-user-images.githubusercontent.com/3358707/384077073-de64ae88-4cf1-44b8-8f05-42a536252309.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY4MDc3NDEsIm5iZiI6MTc1NjgwNzQ0MSwicGF0aCI6Ii8zMzU4NzA3LzM4NDA3NzA3My1kZTY0YWU4OC00Y2YxLTQ0YjgtOGYwNS00MmE1MzYyNTIzMDkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDkwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA5MDJUMTAwNDAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjViZTNmODliYzg2NWQzY2VkODYzMTBlNjA0MDc2MmMxNTI1YTRlMGY5OWZjNjAzZWE1Mzk4MmJhOTU0NjU5MyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.pQdri8B_PFCdDRy_Mv3jIagIVyGRkqfb5cDqgW_jb8o">
> 
> We’d love your feedback on this new experience—try it out and let us know what you think!

**Comment by ghost** [comment]
#2036

**Comment by cptthura-alt** [comment]
https://developers.redhat.com/products/openjdk/getting-started?success=true&tcWhenSigned=January+1%2C+1970&tcWhenEnds=January+1%2C+1970&tcEndsIn=0&tcDuration=365&tcDownloadFileName=java-21-openjdk-21.0.4.0.7-1.win.jdk.x86_64.zip&tcRedirect=5000&tcSrcLink=https%3A%2F%2Fdevelopers.redhat.com%2Fcontent-gateway%2Fcontent%2Forigin%2Ffiles%2Fsha256%2F02%2F028944c88c3e4055e45fd4835400ad03931fc0a3f439d833c800795c9c16502a%2Fjava-21-openjdk-21.0.4.0.7-1.win.jdk.x86_64.zip&p=SPMM%3A+OpenJDK&pv=July+2024&tcDownloadURL=https%3A%2F%2Faccess.cdn.redhat.com%2Fcontent%2Forigin%2Ffiles%2Fsha256%2F02%2F028944c88c3e4055e45fd4835400ad03931fc0a3f439d833c800795c9c16502a%2Fjava-21-openjdk-21.0.4.0.7-1.win.jdk.x86_64.zip%3F_auth_%3D1764835515_c7f9af3f4f4341ae4093cf9e7ec56e42
